### PR TITLE
versions update

### PR DESCRIPTION
**Pull request by antonkulaga**
I updated to latest scalajs version

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import bintray.Keys._
+import BintrayPlugin.autoImport._
```
Comment (vmunier) [inline build.sbt]: You can remove this line. 0.3.0 bintray plugin is an auto plugin and the bintray settings are imported automatically.

**Comment by vmunier** [comment]
Thank you.

**Comment by antonkulaga** [comment]
I deleted redundant import as you asked.

**Comment by vmunier** [comment]
:+1:

**Pull request merged**